src/main/java/com/google/edith/servlets/Receipt.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.edith.servlets;
+
+/** Encapsulate User info and logout url. */
+public final class Receipt {
+  private final String userId;
+  private final String storeName;
+  private final String date;
+  private final String name;
+  private final String fileUrl;
+  private final float totalPrice;
+  private final Item[] items;
+
+  public Receipt(
+      String userId,
+      String storeName,
+      String date,
+      String name,
+      String fileUrl,
+      float totalPrice,
+      Item[] items) {
```

Comment:
This constructor is a little unwieldy and can be error-prone. E.g. userId and storeName could be swapped and things wouldn't break leading to a hard to catch bug.

If you can use an AutoValue builder that would be ideal, otherwise a todo should also be sufficient since we are short on time

Reply:
I added a todo for it.